### DISCUSSION TITLE
Is this code for the plugin of USD3dsMax's official release?

### DISCUSSION THREAD

**Discussion by Pill-bug-net** [Autodesk/3dsmax-usd#10] [Q&A]
Hello guys,
I am new to exploring Max Plugin codes...
Does this GitHub code have differences between the released plugins?
If that is so, what are the differences between them? Such as expanded support and so on.

Thanks!

**Comment by brossee-adsk** [comment]
Hi @Pill-bug-net,
Thank you for your question.

There are no differences between the official released plugin and what is released in the github repo. We are releasing the code part for the public release version of the plugin. The beta version of the plugin is kept private as of today. We might review our github release cycle in the future.

Also, to be transparent, we have kept some elements of the plugin private but those are released in the form of library elements (UFE, Python, MaterialX 3ds Max material plugin). Further details are part of the [build instructions](https://github.com/Autodesk/3dsmax-usd/blob/dev/doc/build.md#3-universal-front-end-ufe).

I hope this answers your questions.

**Reply by Pill-bug-net** [reply]
Hi @brossee-adsk 
Thank you for the info!!
